### ISSUE TITLE
Assigning multiple styles to the same cell in separate calls not possible

### ISSUE THREAD

**Issue by palatinuse** [cbailiss/basictabler#4]
Assigning a style to the same cell more than once is not possible, and yields the following error message:

> Error in cell$setPropertyValues(declarations) : 
  attempt to apply non-function 

Minimal reproducable example:

```{r}
tbl <- BasicTable$new()
tbl$addData(data.frame(a = c(1)), columnNamesAsColumnHeaders = F)
tbl$setStyling(
  declarations = list("font-weight" = "bold"),
  cells = tbl$getCells(specifyCellsAsList = FALSE, rowNumbers = c(1), columnNumbers = c(1)))
tbl$setStyling(
  declarations = list("text-align" = "center"),
  cells = tbl$getCells(specifyCellsAsList = FALSE, rowNumbers = c(1), columnNumbers = c(1)))
tbl$renderTable()
```

Of course, one can overcome this by gathering all styles to be applied to a given cell, and apply it in a single call. However, it would be desirable to be able to do this in multiple runs as well.

Work-around:

```{r}
tbl <- BasicTable$new()
tbl$addData(data.frame(a = c(1)), columnNamesAsColumnHeaders = F)
tbl$setStyling(
  declarations = list("font-weight" = "bold", "text-align" = "center"),
  cells = tbl$getCells(specifyCellsAsList = FALSE, rowNumbers = c(1), columnNumbers = c(1)))
tbl$renderTable()
```

**Comment by cbailiss** [comment]
Thanks for the report and sorry for the delayed acknowledgement.
This is definitely a bug.  
I will aim to fix and get an updated version on CRAN within the next week.

**Comment by cbailiss** [comment]
As an aside, there is also a slightly simpler way to set styling that eliminates the need to call `tbl$getCells()`:

`tbl$setStyling(1, 1, declarations = list("font-weight" = "bold"))`

This was added in the last release.  It is described in the vignettes, but not very prominently.  I'll make it a bit clearer in the next release.

**Comment by cbailiss** [comment]
Fixed in commit 456f6b91 .

**Comment by cbailiss** [comment]
Fixed in version v0.3.0 now available on CRAN.
